Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+\section{Processi di supporto}
+\subsection{Processo di documentazione}
+\subsubsection{Scopo}
+L'obiettivo proposto dalla sezione è quello di prendere nota delle norme che andranno a regolare la stesura della documentazione durante il ciclo di vita del software, così da ottenere documenti formalmente concordi.
```

Comment:
Cambiare
"L'obiettivo proposto dalla sezione è quello di prendere nota delle norme che andranno a regolare la stesura della documentazione durante il ciclo di vita del software ..."
in
"... L'obiettivo della sezione è quello di normare le attività di stesura e aggiornamento della documentazione svolte durante il ciclo di vita del software ..."

Reply:
Fixed